asv_bench/asv.conf.json

```diff
@@ -41,7 +41,7 @@
     // pip (with all the conda available packages installed first,
     // followed by the pip installed packages).
     "matrix": {
-        "numpy": [],
+        "numpy": ["1.23.5"],  // https://github.com/pandas-dev/pandas/pull/50356
```

Comment:
asv doesn't allow pinning with `<`

Reply:
Yeah I'd just go with the newest version as well.